src/components/themed/TransactionListTop.js

```diff
@@ -142,7 +142,7 @@ export class TransactionListTopComponent extends React.PureComponent<Props, Stat
     const styles = getStyles(theme)
 
     const isStakingPolicyAvailable =
-      stakePolicies != null &&
+      stakePolicies.length > 0 &&
```

Comment:
Line 151 may need some TLC. Also skim through to all occurrences of `stakePolicies` for other null-checks and replace them with `.length` check.